Auth0/Auth0Authentication.swift

```diff
@@ -176,6 +176,16 @@ struct Auth0Authentication: Authentication {
         return Request(session: session, url: oauthToken, method: "POST", handle: authenticationObject, payload: payload, logger: self.logger, telemetry: self.telemetry)
     }
 
+    func revoke(refreshToken: String) -> Request<Void, AuthenticationError> {
+        let payload: [String: Any] = [
+            "tokenEndpointAuthMethod": "none",
```

Comment:
I based on https://auth0.com/docs/tokens/preview/refresh-token#revoke-a-refresh-token
grant_type not mentioned either.